keras_nlp/src/models/vgg/vgg_backbone_test.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+
+from keras_nlp.src.models.vgg.vgg_backbone import VGGBackbone
+from keras_nlp.src.tests.test_case import TestCase
+
+
+class VGGBackboneTest(TestCase):
+    def setUp(self):
+        self.init_kwargs = {
+            "stackwise_num_repeats": [2, 2, 3, 3, 3],
```

Comment:
In general, we should test with small inputs. Just to keep our test time down. Tests slowness is going to be a real problem as we combine the cv models. E.g. something like this...

```
"stackwise_num_repeats": [1, 2, 3],
"stackwise_number_filters": [4, 8, 16],
"input_shape": (16, 16, 3),
```

We want to test the "interesting logic". E.g. different filters sizes, block repeats, etc. But keep our actual test arrays very small.